src/rust/ide/src/controller/graph.rs

```diff
@@ -729,6 +731,37 @@ impl Handle {
         Ok(node_info.id())
     }
 
+    /// Adds an example to the graph.
+    ///
+    /// The example piece of code will be inserted as a new function definition, and in current
+    /// graph the node will appear which calls this function.
```

Comment:
"graph the node will appear which calls this function." -> "graph a node calling this function will appear."